example/proxy_conf.yaml

```diff
@@ -67,22 +59,30 @@ LOGGING:
       level: DEBUG
       formatter: simple
       stream: ext://sys.stdout
-    saml2_debug_file:
-      class: logging.handlers.RotatingFileHandler
-      level: DEBUG
-      formatter: simple
-      filename: logs/saml2_debug.log
-      maxBytes: 104857600 # 100MB
-      backupCount: 20
-      encoding: utf8
-    oidcop_debug_file:
-      class: logging.handlers.RotatingFileHandler
-      level: DEBUG
-      formatter: simple
-      filename: logs/oidcop_debug.log
-      maxBytes: 104857600 # 100MB
-      backupCount: 20
-      encoding: utf8
+    # spid_daily:
+    #   class: logging.handlers.TimedRotatingFileHandler
+    #   level: INFO
+    #   formatter: simple
+    #   filename: logs/spid.log
+    #   when: D
+    #   interval: 1
+    #   backupCount: 860
```

Comment:
why?

let's keep cie and spid policy retention configured by default

Reply:
In docker spid daily is not used, the log are sended to docker logs from STDOUT. Spid daily handler is used only for local installation and must be configured manually in proxy_conf.yaml.
If remain uncommented  the image require an un-necessary write access right to `logs/spid.log` empty file.

If someone want use this must uncommented this and change the handler loggers from `[console]` to `[spid_daily]`. If you want to use spid-daily you still need to edit the proxy-conf file.